crowbar_framework/config/locales/updater/en.yml

```diff
@@ -15,6 +15,9 @@
 #
 
 en:
+  nav:
+    utils:
+      upgrade: 'Prepare nodes for upgrade' 
```

Comment:
This title should be shorter. As this action updates the proposals this should be a post action with a confirmation.